device-types/Sophos/XGS-3100.yaml

```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Sophos
+model: XGS 3100
+slug: sophos-xgs-3100
+u_height: 1
+weight: 4.7
+weight_unit: kg
+is_full_depth: false
+airflow: front-to-rear
+console-ports:
+  - name: COM Serial
+    type: rj-45
+  - name: COM USB
+    type: usb-micro-b
+  - name: Front USB 1
+    type: usb-a
+  - name: Front USB 2
+    type: usb-a
+  - name: Rear USB
+    type: usb-a
```

Comment:
Are these USBs used for console access? If not, please remove them.

Reply:
removed as requested, I was unsure whether to add them or not....